tests/test_recipient_csv.py

```diff
@@ -430,6 +430,22 @@ def test_overly_big_list():
     assert list(big_csv.rows_with_message_too_long) == []
 
 
+def test_big_empty_list():
+    process = Mock()
+
+    lots_of_commas = ',' * 100_000
```

Comment:
Can we take a similar approach to 5a730be here? This test takes about 6s to run, which I'd like to avoid if possible.

Reply:
I‘ve changed the test in 2435dca

Reply:
Thanks. You could go smaller still e.g. just 2 commas would be sufficient for the new assertion. However, testing locally indicates the difference is now minimal.